var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -23,7 +23,8 @@ class Neuron(Package):
     git      = "https://github.com/nrnhines/nrn.git"
 
     version('develop', branch='master')
-    version('2018-10', commit='b3097b7', preferred=True)
+    version('7.6.6',   tag='7.6.6', preferred=True)
```

Comment:
Feel free to add 7.7 as well (not setting preferred).
For 7.6.6 I would not set as preferred right away. We change that when we see all tests run well.
You can drop 2018-09, was almost never used.

Reply:
I added only version 7.6.6, as this is the most recent one and the one Michael Hines suggested using